scripts/insert_into_appointments.sql

```diff
@@ -1 +1,28 @@
--- insert appointments creating connections between doctors and patients
+INSERT INTO appointments(app_date, app_time)
+  SELECT 'August 12, 2016', '10am'
+  FROM doctors d, patients p
+  WHERE p.given_name = 'Marsha'
+  AND p.surname = 'Wilcox'
+  AND d.given_name = 'Vicky'
+  AND d.surname = 'Hilton'
+  UNION
+  SELECT 'August 17, 2016', '1pm'
+  FROM doctors d, patients p
+  WHERE p.given_name = 'Ivory'
+  AND p.surname = 'Talley'
+  AND d.given_name = 'Spencer'
+  AND d.surname = 'Hubbard'
+  UNION
+  SELECT 'August 12, 2016', '10am'
+  FROM doctors d, patients p
+  WHERE p.given_name = 'Marsha'
+  AND p.surname = 'Wilcox'
+  AND d.given_name = 'Spencer'
+  AND d.surname = 'Hubbard'
+  UNION
+  SELECT 'August 17, 2016', '1pm'
+  FROM doctors d, patients p
+  WHERE p.given_name = 'Blake'
+  AND p.surname = 'Hodge'
+  AND d.given_name = 'Vicky'
+  AND d.surname = 'Hilton'
```

Comment:
Close...but no cigar. 

1) you always need a semicolon at the end
2) you also need some commas. look at the code you wrote in class
3) look at your 1st line. What are the parameters you require for inserting into the appointments table? only date and time, for which you provide string literals in the `SELECT` statement. If those fields are all that you require, and you already provide that, then why do you even need to other stuff?? 

Take a look at this solution:
```sql
INSERT INTO appointments(date, time, doctor_id, patient_id)
  SELECT '20150812', '1000',
    (SELECT id FROM doctors WHERE surname = 'Hilton'),
    (SELECT id FROM patients WHERE surname = 'Wilcox')
  UNION
  SELECT '20150817', '1300',
    (SELECT id FROM doctors WHERE surname = 'Hubbard'),
    (SELECT id FROM patients WHERE surname = 'Talley')
  UNION
  SELECT '20150817', '1000',
    (SELECT id FROM doctors WHERE surname = 'Hubbard'),
    (SELECT id FROM patients WHERE surname = 'Wilcox')
  UNION
  SELECT '20150817', '1300',
   (SELECT id FROM doctors WHERE surname = 'Hilton'),
   (SELECT id FROM patients WHERE surname = 'Hodge')
;
```